espnet2/asr/postencoder/hugging_face_transformers_postencoder.py

```diff
@@ -48,12 +51,32 @@ def __init__(
         else:
             self.transformer = model
 
-        if hasattr(self.transformer, "embed_tokens"):
```

Comment:
Should not we still remove these attributes from self.transformer even when lan_token_id is -1.